### DISCUSSION TITLE
Web Browser Oscilloscope - Trigger Level Settings

### DISCUSSION THREAD

**Discussion by tux-friend** [ozel/DIY_particle_detector#22] [Q&A] [answered]
Hello there!

I really like this project, great work! I've built one electron-detector and now try to figure out if it really works... 
As I have no oscilloscope by hand I used a BNC to 3.5mm cable and a soundcard to use the WebGui Oscilloscope.
Comparing to the pictures on the wiki my signal seems to be a bit different. When I turn the electron detector on, the signal changes rapidly for a about 10-20 seconds and then seems to be frozen in.
![Bildschirmfoto von 2020-12-14 15-02-29](https://user-images.githubusercontent.com/4411054/102091832-9e8ed080-3e1f-11eb-91d3-4bfb50bcdb65.png)
![Bildschirmfoto von 2020-12-14 15-02-35](https://user-images.githubusercontent.com/4411054/102091834-9f276700-3e1f-11eb-8662-db419dd5e019.png)
And also shouldn't the curve be much more oscillating - broader noise?

**Comment by ozel** [comment, marked as answer]
Hi Felix, 
glad you like it!

This behaviour is normal and a good sign. The signal gets smaller in the first few seconds after switching on because some capacitances are discharged. The same happens directly after closing the case, since the electronics got "charged" by the ambient light.
Your second picture could be already a very small but real pulse, but it should be a bit wider to be sure.

It is normal that the picture freezes, it always shows you the last signal that was larger than the red treshhold line. This is how a real  oscilloscope works as well when set to "normal" trigger mode.
Now in order to see the signal again after switching on, you need to move the trigger upwards either by clicking on the UP arrow button or by using the cursors keys. The red trigger line must be set somewhat below the noise range - not too close not too far away - such that you see only proper pusles.

Since every soundcard has different sensitvity, there is no universal trigger level that works for every audio hardware.
The default level of -8000 for the elctron detector works usually OK with headset connectors, for example on iPads and iPhones, but often the trigger level needs to be adjusted to the audio hardware that one uses.
On a computer/laptop it also depends on the microphone volume settings of the operating system. Usually values between 50 and 100% are good, you should try out different settings to get the best pulses versus noise.
More on one specific low-cost soundard that I recommend in particular for the alpha-sectrometer: https://github.com/ozel/DIY_particle_detector/wiki/Soundcards

Below are two example pulses recorded with the recent version of the web oscilloscope, on an iPhone via the headset connector. 

![small Electron-Detector pulse on iPad](https://user-images.githubusercontent.com/653212/102107670-43b2a480-3e32-11eb-8bed-e6d58cc27efb.png)

The second picture shows a strong oscoillation, because this pulse is a bit too large for the iPad. Some headset inputs will react like that, real soundcards have a wider input range and usually do not show these oscillations within one pulse.  

![big Electron Detector pulse on iPad with oscillations](https://user-images.githubusercontent.com/653212/102107882-85434f80-3e32-11eb-8e6c-f4cb3436e554.png)

I will update the wiki with those pictures.
Oliver

**Reply by ozel** [reply]
And I hope you have something radioactive in front of the detector. Otherwise it may take several minutes until a cosmic ray or radon decay particle hits the detector.
